src/shared/api/types/activity.ts

```diff
@@ -23,3 +23,49 @@ export type ActivityDTO = {
 }
 
 export type ActivityItemDetailsDTO = TextItemDTO | CheckboxItemDTO | RadioItemDTO | SliderItemDTO | SelectorItemDTO
+
+export type AnswerPayload = {
+  appletId: string
+  version: string
+  flowId: string | null
+  activityId: string
+  answers: Array<AnswerTypesPayload>
+}
+
+export type AnswerTypesPayload =
+  | TextAnswerPayload
+  | MultiSelectAnswerPayload
+  | SingleSelectAnswerPayload
+  | SliderAnswerPayload
+
+export type TextAnswerPayload = {
+  activityItemId: string
+  answer: {
+    value: string
+    shouldIdentifyResponse: boolean
+  }
+}
+
+export type MultiSelectAnswerPayload = {
+  activityItemId: string
+  answer: {
+    value: Array<string> // Array of IDs
```

Comment:
Please create a shared type and use it so that you avoid putting these comments:
`type ID = string`;